index.js

```diff
@@ -1,88 +1,130 @@
 // ***** Question 1 *****
 // *** Uncomment the lines below to test
-// console.log("%cQuestion 1", "color: red") 
-
-// console.log(counter) 
-// // => 2
-// console.log("%c----------", "color: red") 
+console.log("%cQuestion 1", "color: red") 
+let counter = 1
+counter = 2
+console.log(counter) 
+// => 2
+console.log("%c----------", "color: red") 
 
 
 // ***** Question 2 *****
 // *** Uncomment the lines below to test
 // *** after testing, comment the line causing the error back in 
 // *** otherwise, the error will stop the rest of your code from running
-// console.log("%cQuestion 2", "color: red") 
-
-// console.log(name)
+console.log("%cQuestion 2", "color: red") 
+const name = "Diana"
+console.log(name)
 // => "Raffy"
 // name = "Not Raffy" 
 // => TypeError
-// console.log("%c----------", "color: red") 
+console.log("%c----------", "color: red") 
 
 
 
 // ***** Question 3 *****
 // *** Uncomment the lines below to test
-// console.log("%cQuestion 3", "color: red") 
+console.log("%cQuestion 3", "color: red") 
+
+const drinkWater = currentThirstLevel => {
+  console.log("Man I sure am thirsty")
+  currentThirstLevel --
+  console.log("Ahh that hits the spot")
+  return currentThirstLevel
+}
 
-// console.log(drinkWater(12))
-// // => "Man I sure am thirsty"
-// // => "Ahh that hits the spot"
-// // => 11
-// console.log("%c----------", "color: red") 
+console.log(drinkWater(12))
+// => "Man I sure am thirsty"
+// => "Ahh that hits the spot"
+// => 11
+console.log("%c----------", "color: red") 
 
 
 
 // ***** Question 4 *****
 // *** Uncomment the lines below to test
-// console.log("%cQuestion 4", "color: red") 
+console.log("%cQuestion 4", "color: red") 
+
+
+const sameSameButDifferent = (num, maybeNum) => {
+  if (num === maybeNum) {return "same same"}
+  else if (num == maybeNum) {return "same same (but different)"}
+  else {return "different"}
+}
 
```

Comment:
Nice! The key here was understanding the difference between `===` and `==`. You could also convert the string to a number with parseInt to avoid `==`

PS love the single line `if` return blocks - definitely no need to make a new line for a one line return! If you're using a one liner, you also don't need the `{ }` brackets, btw!